### PR TITLE
Ft-Create-Login-Page

### PR DESCRIPTION
**Pull request by Morsetim**
https://trello.com/c/K2VeOgvY/21-login-page

### CODE REVIEW

File: src/pages/SignIn/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    return(
+        <div className="main">
+            <div className="signin-container">
+                <div className="signup-div">
+                    <div className="empty-div"></div><div className="signup"><p>No Account?</p><p id="signup">Sign Up</p></div>
+                </div>
+                <div>
+                <FormTitle text="Login" rectColor="none" className="form-title" textColor="black"/>
+                <hr />
```
Comment (hnobi) [inline src/pages/SignIn/index.js:31]: Please remove hr and make FormTitle `rectColor`

File: src/pages/SignIn/index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    }
+
+    return(
+        <div className="main">
+            <div className="signin-container">
+                <div className="signup-div">
+                    <div className="empty-div"></div><div className="signup"><p>No Account?</p><p id="signup">Sign Up</p></div>
+                </div>
+                <div>
+                <FormTitle text="Login" rectColor="none" className="form-title" textColor="black"/>
```
Comment (hnobi) [inline src/pages/SignIn/index.js:30]: add a rectColor

**Review by hnobi — CHANGES_REQUESTED** [review]
Please the Login should look something like this below. 

Screenshot 2021-11-10 at 11 55 18 PM

File: src/pages/SignIn/index.js (hunk 1 above)
Comment (Morsetim) [reply src/pages/SignIn/index.js:31]: Oh I see, will be get it done as soon as possible

File: src/pages/SignIn/index.js (hunk 2 above)
Comment (Morsetim) [reply src/pages/SignIn/index.js:30]: Okay

**Review by hnobi — APPROVED** [review]

**Pull request merged**